test/specs/scale.category.tests.js

```diff
@@ -136,6 +136,37 @@ describe('Category scale tests', function() {
 		expect(scale.ticks).toEqual(mockData.yLabels);
 	});
 
+	it('Should generate ticks from the axis labels', function() {
```

Comment:
Mock data and manual construction of the scale are not the most trustful way to setup unit tests. I think we should instantiate a real chart if that's possible (which should be almost always the case).

```javascript
var chart = window.acquireChart({
    type: 'line',
    data: {
        data: [10, 5, 0, 25, 78]
    },
    options: {
        scales: {
            xAxes: [{
                id: 'x',
                type: 'category',
                labels: ['tick1', 'tick2', 'tick3', 'tick4', 'tick5']
            }]
        }
    }
});

var scale = chart.scales.x;
expect(scale.ticks).toEqual(['tick1', 'tick2', 'tick3', 'tick4', 'tick5']);
```

It's even less code and more robust since you don't manually manipulate the scale to obtain the values to check.